website/versioned_docs/version-0.79/_getting-started-linux-android.md

```diff
@@ -12,6 +12,11 @@ Follow the [installation instructions for your Linux distribution](https://nodej
 
 React Native currently recommends version 17 of the Java SE Development Kit (JDK). You may encounter problems using higher JDK versions. You may download and install [OpenJDK](https://openjdk.java.net) from [AdoptOpenJDK](https://adoptopenjdk.net/) or your system packager.
 
+```shell
+sudo apt update
+sudo apt install temurin-17-jdk
+```
```

Comment:
This feels like out of context + we haven't tested this workflow to be able to recommend it